src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -624,6 +624,22 @@ public async Task<bool> RecordFinancialGrade(Guid applicationId, FinancialReview
             return true;
         }
 
+        public async Task<bool> UpdateFinancialReviewDetails(Guid applicationId, FinancialReviewDetails financialReviewDetails)
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+                {
+                    await connection.ExecuteAsync(@"UPDATE Apply 
+                                                         SET FinancialGrade = @financialReviewDetails
```

Comment:
This probably should have a check to ensure it's not a Deleted application (check has been used in other repo/sql queries)